src/js/components/DataTable/DataTable.js

```diff
@@ -236,6 +242,16 @@ const DataTable = ({
     else direction = 'asc';
     const nextSort = { property, direction, external };
     setSort(nextSort);
+    if (onUpdate) {
+      onUpdate({
+        expanded: Object.keys(groupState).filter(
```

Comment:
Since we do this several times, it might be a little cleaner to create a helper function, to enable these three lines to be one where it's needed.
```
const filterExpandedGroups = (groupState) => groupState.filter(...);
```
just a thought, not required for approval.

Reply:
I think because it's so simple and sometimes its groupState and sometimes nextGroupState I might leave it if that's cool.